packages/manager/apps/pci-rancher/src/pages/layout.tsx

```diff
@@ -1,11 +1,20 @@
-import React, { useContext, useEffect } from 'react';
-import { Outlet, useLocation } from 'react-router-dom';
 import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { defineCurrentPage } from '@ovh-ux/request-tagger';
+import React, { useContext, useEffect } from 'react';
+import { Outlet, useLocation, useMatches } from 'react-router-dom';
 
 function RoutingSynchronisation() {
   const location = useLocation();
   const { shell } = useContext(ShellContext);
 
+  const matches = useMatches();
+
+  useEffect(() => {
+    const match = matches.slice(-1);
```

Comment:
**nitpick:** Since we only need first element, better to restructure for better readability.
```suggestion
    const [match] = matches.slice(-1);
```